tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -591,6 +591,8 @@ class CompilerTest {
   TEST_METHOD(CodeGenResPhi)
   TEST_METHOD(CodeGenResPhi2)
   TEST_METHOD(CodeGenRootSigEntry)
+  TEST_METHOD(CodeGenRootSigProfile)
+  TEST_METHOD(CodeGenRootSigProfile2)
```

Comment:
Should add tests for the two negative cases we support:

1. Root signature macro name is not found.
2. Root signature in macro is invalid.